lib/FCM/System/Make/Share/Config.pm

```diff
@@ -217,6 +219,18 @@ sub _parse_name {
     $m_ctx->set_name($entry->get_value());
 }
 
+# Parse "require-version" declaration from a config entry.
+sub _parse_require_version {
+    my ($attrib_ref, $m_ctx, $entry) = @_;
+    my ($version) = shellwords($attrib_ref->{util}->version());
+    my ($min_version, $max_version) = shellwords($entry->get_value());
+    if (    $min_version gt $version
+        ||  defined($max_version) && $version gt $max_version
+    ) {
```

Comment:
Just to make sure I understand, is this doing a text comparison on the strings (rather than numeric, as 2015.12.0 isn't a number)? And if one value is set as require-version it's taken as a minimum, and if there are two the first is a minimum and the second is a maximum?